Resources/config/security.xml

```diff
@@ -10,6 +10,13 @@
     </parameters>
     <services>
         <service id="sonata.admin.security.handler.noop" class="%sonata.admin.security.handler.noop.class%" public="false"/>
+        <service id="sonata.admin.security.handler.cache"
+            class="Sonata\AdminBundle\Security\Handler\CacheSecurityHandler"
+            decorates="sonata.admin.security.handler"
```

Comment:
Sadly I can't use this feature introduced in Symfony 2.5. Can I directly set the `sonata.admin.security.handler` to be the cache one and use the `security.handler` configuration value as first argument?